### DISCUSSION TITLE
JJUI: A TUI for interacting with `jj`

### DISCUSSION THREAD

**Discussion by idursun** [jj-vcs/jj#4608] [Show and tell]
Hi folks,

I made a TUI for interacting with `jj` for my personal needs and thought it might be useful for folks who are looking for something in that space. I know there are many other GUI/TUIs built for `jj`, (and they are all great) but this one just fits in the way I use `jj`.

https://github.com/idursun/jjui

I'd say the main difference is that all of the operations are done around the tree view of the revisions. 

![GIF](https://github.com/idursun/jjui/raw/main/docs/jjui_rebase.gif)

**Comment by ocharles** [comment]
I think you forgot to link us to it! :smile:

**Reply by idursun** [reply]
🤦🏿‍♂️ I did it just now :)

**Comment by romaia** [comment]
This is really cool. Thanks a lot.

It completely fixes my need for interactive rebases

**Comment by nickchomey** [comment]
jj is already incredible and jjui makes it completely revolutionary. It's such a pleasure to use. 

Moreover, @idursun is one of the most responsive and receptive maintainers I've ever come across.

Start using jjui!
